platform-sdk/docs/proposals/TSS-Library/TSS-Library.md

```diff
@@ -334,7 +334,6 @@ Multiple signatures can be aggregated to create an aggregate `TssSignature`. An
 #### Security Considerations
 As long as an adversary Participant knows fewer than a threshold number of decryption keys, they cannot recover enough information to start forging threshold signatures.
 Adversarial participants may learn the shares of the participants whose keys they have compromised, but more is needed to recover the secret.
-Suggested change
 For security, adversarial parties are free to choose low or zero entropy values for protocol inputs such as shareIds.
 There will still be sufficient entropy in the entire protocol from the honest nodes.
 For security, adversarial parties might choose low or zero entropy values for protocol inputs such as shareIds, but here will still be sufficient entropy in the overall protocol from honest nodes.
```

Comment:
Still duplicating lines; I think the first two are the old version, the last line is the revision.